packages/unity-bootstrap-theme/src/scss/extends/_cards.scss

```diff
@@ -519,28 +519,37 @@ Cards - Table of Contents
   background-color: transparent;
 }
 
-.card-foldable {
+.accordion-item {
   border-color: $uds-color-base-gray-3;
   border-left: $uds-size-spacing-1 solid $uds-color-base-gold;
   height: auto;
 
-  &.card-maroon {
+  &:first-of-type {
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+  }
+  &:last-of-type {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
```

Comment:
Instead of adding these links

Lets override the BS5 sass variable 

Create new file: `packages/unity-bootstrap-theme/src/scss/variables/_accordion.scss`
```scss
// Bootstrap accordion variables to override.
$accordion-border-radius: 0;

// There are variables to override, but we just build on top of them.

``` 

Add to file `packages/unity-bootstrap-theme/src/scss/_unity-bootstrap-theme-variables.scss`
```scss
@import "variables/accordion";
...
...
...
```